src/wp-includes/block-bindings/post-meta.php

```diff
@@ -51,6 +48,7 @@ function _register_block_bindings_post_meta_source() {
 		array(
 			'label'              => _x( 'Post Meta', 'block bindings source' ),
 			'get_value_callback' => '_block_bindings_post_meta_get_value',
+			'uses_context'       => array( 'postId', 'postType' ),
```

Comment:
Is `postType` ever used on the client? I don't see it being referenced on the server.

Reply:
Yes, it is used to retrieve the custom field depending on the `postType`: [link](https://github.com/WordPress/gutenberg/blob/06cfe8c95015189c762afd73acd31c3c5c76ee17/packages/editor/src/bindings/post-meta.js#L22-L27).